plugins/home/src/components/CustomHomepageGrid.tsx

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useCallback, useMemo } from 'react';
+import { Layout, Layouts, Responsive, WidthProvider } from 'react-grid-layout';
+import {
+  componentDataKey,
+  MaybeComponentNode,
+  storageApiRef,
+  useApi,
+} from '@backstage/core-plugin-api';
+import 'react-grid-layout/css/styles.css';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import 'react-resizable/css/styles.css';
+import {
+  createStyles,
+  ListItemSecondaryAction,
+  makeStyles,
+  Modal,
+  Theme,
+} from '@material-ui/core';
+import ListItemText from '@material-ui/core/ListItemText';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import { compact } from 'lodash';
+import IconButton from '@material-ui/core/IconButton';
+import AddIcon from '@material-ui/icons/Add';
+import { useObservable } from 'react-use';
```

Comment:
try `import useObservable from 'react-use/lib/useObservable';`

Reply:
Ah true, totally missed that. Thanks!